pkg/services/object/acl/acl.go

```diff
@@ -459,8 +460,12 @@ func (b Service) findRequestInfo(
 		return info, ErrUnknownRole
 	}
 
-	// find verb from token if it is present
-	verb := sourceVerbOfRequest(req, op)
+	// Check if session token verb is equal to the current operation.
+	// Unknown means everything is allowed.
```

Comment:
Also, can you fix API spec and mention such behavior [there](https://github.com/nspcc-dev/neofs-api/blob/650c367529c6fe6b215ea8d5f183b07b4ac8f9a9/session/types.proto#L40)?